debian/changelog

```diff
@@ -1,3 +1,9 @@
+pinball-table-gnu (0.0.20200601-3) UNRELEASED; urgency=medium
```

Comment:
i am planning to make a new upstream release 
would you like to hold change on this cl file ?

Reply:
Ah, ack - I've dropped this change; I wasn't sure whether you preferred changelog entries or not.